config/v1/types_apiserver.go

```diff
@@ -0,0 +1,67 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// APIServer holds cluster-wide information about api-servers.  The canonical name is `cluster`
+type APIServer struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              APIServerSpec   `json:"spec"`
+	Status            APIServerStatus `json:"status"`
+}
+
+type APIServerSpec struct {
+	// servingCert is the TLS cert info for serving secure traffic.
+	// +optional
+	ServingCerts APIServerServingCerts `json:"servingCerts,omitempty"`
+	// clientCA is the certificate bundle for the signers that will be recognized for incoming client certificates.
+	// The namespace for this config map is openshift-config.
+	// +optional
+	ClientCA ConfigMapNameReference `json:"clientCA,omitempty"`
+}
+
+type APIServerServingCerts struct {
+	// default references a kubernetes.io/tls type secret containing the default TLS cert info for serving secure traffic.
+	// If no named certificates match the server name as understood by a client, this default certificate will be used.
+	// The secret must exist in the openshift-config namespace and contain the following required fields:
+	// - Secret.Data["tls.key"] - TLS private key.
+	// - Secret.Data["tls.crt"] - TLS certificate.
+	// +optional
+	Default SecretNameReference `json:"default,omitempty"`
```

Comment:
This is the name that @mfojtik asked to change.  `defaultServingCertificate` I think